src/modules/m_cap.h

```diff
@@ -35,6 +37,7 @@ class CapEvent : public Event
 	CapEventType type;
 	std::vector<std::string> wanted;
 	std::vector<std::string> ack;
+	std::map<GenericCap*, int> changed; // HACK: clean this up before 2.2
```

Comment:
There's no need for a map, a `std::vector<std::pair<GenericCap*, bool> >` (or int) is enough.